manifests/redhat.pp

```diff
@@ -14,12 +14,22 @@
 
   if $manage_repo {
 
-    $keys = [
+    $keys_src = [
         'https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public',
-        'https://keys.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public',
         'https://keys.datadoghq.com/DATADOG_RPM_KEY_FD4BF915.public',
         'https://keys.datadoghq.com/DATADOG_RPM_KEY_B01082D3.public',
+        'https://keys.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public',
     ]
+    
+    if $agent_version =~ /([0-9]+:)?([0-9]+)\.([0-9]+)\.([0-9]+)((?:~|-)[^0-9\s-]+[^-\s]*)?(?:-([0-9]+))?/ or $agent_version == 'latest' {
```

Comment:
Could you add a comment of what this is supposed to match? :sweat_smile:

Reply:
Yea copy pasted it from another puppet file I will copy paste the comment 👍